Makefile

```diff
@@ -1,17 +1,17 @@
-BASETAG=bitjourney/elasticsearch-ci
-TIMESTAMP=`date "+%Y%m%d%H%M"`
-IMAGETAG=$(BASETAG):$(TIMESTAMP)
-LATESTTAG=$(BASETAG):latest
+BASE_TAG:=bitjourney/elasticsearch-ci
+TIMESTAMP:=`date "+%Y%m%d%H%M"`
+IMAGE_TAG:=$(BASE_TAG):$(TIMESTAMP)
+LATEST_TAG:=$(BASE_TAG):latest
```

Comment:
`=` is lazy calling in Makefile. It isn't necessary lazy calling here, so use `:=`.
see: http://qiita.com/takuyachallenge/items/c73b1d2db3761b931109#%E5%8D%98%E7%B4%94%E5%B1%95%E9%96%8B%E5%A4%89%E6%95%B0